README.md

```diff
@@ -34,8 +34,6 @@ vars:
     pinterest_database: your_schema_name 
 ```
 
-For additional configurations for the source models, please visit the [Pinterest Ads source package](https://github.com/fivetran/dbt_pinterest_source).
```

Comment:
^ It looks like this content was accidentally copied from the Pinterest Ads modeling package README, so I removed it